dashboard/test/ui/features/curriculum_platform/levelbuilder/lesson_edit_page.feature

```diff
@@ -28,3 +28,38 @@ Feature: Using the Lesson Edit Page
     And element ".uitest-activity-duration-input" has value "15"
 
     And I delete the temp script and lesson
+
+  Scenario: Add a level using the lesson edit page
+    Given I create a levelbuilder named "Levi"
+    And I create a temp script and lesson
+    And I view the temp lesson edit page
+    And I wait until element ".uitest-activity-card" is visible
+    And element ".uitest-open-add-level-button" is visible
+    And element ".uitest-bubble" contains text "1"
+    And element ".uitest-bubble" does not contain text "2"
+
+    # Open the Add Level dialog, search for an artist level and add the first one
+
+    # TODO: use regular click instead once button responds to click events
+    When execute JavaScript expression "$('.uitest-open-add-level-button').simulate('mousedown')"
+    And I wait until element "h2" contains text "Add Levels"
+    And element "#add-level-type" is visible
+    And I select the "Artist" option in dropdown "add-level-type"
+    And element ".fa-search" is visible
+    And I press ".fa-search" using jQuery
+    And I wait until element "td" contains text "Artist"
```

Comment:
What happens if the most recent level in the list is an Artist level? This check won't make sure things loaded right?

Reply:
Good catch! I've updated this with an earlier check to make sure there were no artist levels initially.